src/style.css

```
@@ -1 +1,38 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  gap: 4px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+
+}
+
+.stars--1 :first-child {
```

```suggestion
.stars--1 :first-child,
.stars--2 :nth-child(-n + 2),
.... {
```
Consider using commas for grouping selectors